localtypings/pxtarget.d.ts

```diff
@@ -631,6 +631,7 @@ declare namespace ts.pxtc {
         attributes: CommentAttrs;
         name: string;
         namespace: string;
+        fileName: string;
```

Comment:
@mmoskal , I added tracking of the source file into SymbolInfo so I can do more fine-grain filtering of "shadowed" .ts files in py2ts. See below. I'm not sure this is a sane thing to do with these abstractions. Is this okay?

Reply:
looks good!